src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useContext, useEffect, useState } from 'react';
+import cn from 'classnames';
+import { TodosContext } from '../TodosContext/TodosContext';
+
+export const ErrorNotification: React.FC = () => {
+  const { errorMessage } = useContext(TodosContext);
+  const [isHidden, setIsHidden] = useState(true);
+
+  useEffect(() => {
+    if (errorMessage) {
+      setIsHidden(false);
+    } else {
+      setIsHidden(true);
```

Comment:
You can use ternary operator or just simplify it as follow:
`setIsHidden(!errorMessage)`